Mergin/utils.py

```diff
@@ -1497,7 +1497,7 @@ def setup_tracking_layer(layer: QgsVectorLayer):
     layer.setDefaultValueDefinition(idx, user_default)
 
     layer.setDisplayExpression(
-        "\"tracked_by\" ||' on '|| format_date( \"tracking_end_time\", 'dd MMM yyyy') ||' at '|| format_date(\"tracking_end_time\", 'H:MM')"
+        "\"tracked_by\" ||' on '|| format_date( \"tracking_end_time\", 'dd MMM yyyy') ||' at '|| format_date(\"tracking_end_time\", 'H:mm')"
```

Comment:
It doesn't seem to be documented in `format_date()`, but this:
```suggestion
        "\"tracked_by\" ||' on '|| format_date( \"tracking_end_time\", 'dd MMM yyyy') ||' at '|| format_date(\"tracking_end_time\", 'H:mm (t)')"
```
should add the timezone, so it would look like:
```
stefanos on 30 May 2025 at 22:59 (EEST)
```